sdk/python/tests/integration/registration/test_cli.py

```diff
@@ -128,6 +141,56 @@ def make_feature_store_yaml(project, test_repo_config, repo_dir_name: PosixPath)
     return yaml.safe_dump(config_dict)
 
 
+NULLABLE_ONLINE_STORE_CONFIGS: List[IntegrationTestRepoConfig] = [
+    IntegrationTestRepoConfig(
+        provider="local",
+        offline_store_creator=FileDataSourceCreator,
+        online_store=None,
+    ),
+]
+
+if os.getenv("FEAST_IS_LOCAL_TEST", "False") == "True":
```

Comment:
Can somebody explain the use case for `FEAST_IS_LOCAL_TEST` to me please?

Reply:
`FEAST_IS_LOCAL_TEST` means the test only uses the local provider, so it doesn't require cloud resources. We mainly use it in `repo_configuration.py`: if `FEAST_IS_LOCAL_TEST=True`, we don't use all the `IntegrationTestRepoConfigs`, only the one that uses the local provider.